### PR TITLE
9424 Refactoring act specs.

### PR DESCRIPTION
**Pull request by simon2k**

### CODE REVIEW

File: lib/kameleon/dsl/act.rb [hunk 1]
```diff
@@ -39,37 +39,23 @@ def click(*links)
       def fill_in(fields)
         fields.each_pair do |value, selector|
           case value
-            when :check
+            when :check, :choose, :uncheck
               one_or_all(selector).each do |locator|
-                session.check locator
+                session.send(value, locator) if modifiable_locator?(locator)
```
Comment (cs3b) [inline lib/kameleon/dsl/act.rb]: nice refactoring :-)

we could move 

```
if modifiable_locator?(locator)
```

above case, and raise proper exception when trying to modify readonly or disabled field, what you think?

File: lib/kameleon/dsl/see.rb [hunk 2]
```diff
@@ -19,16 +19,24 @@ def see(*content)
                     when :empty
                       one_or_all(locator).each do |selector|
                         session.should rspec_world.have_field(selector)
-                        session.find_field(selector).value.should == ""
+                        session.find_field(selector).value.to_s.should == ""
```
Comment (cs3b) [inline lib/kameleon/dsl/see.rb:22]: what is the case when value is not a string ?

Comment (simon2k) [inline lib/kameleon/dsl/see.rb:22]: If value is not string - value is nil.
I convert this into string and I've got empty string - "".

File: lib/kameleon/dsl/act.rb (hunk 1 above)
Comment (simon2k) [inline lib/kameleon/dsl/act.rb]: I have moved this, but without adding raising exception.

**Pull request merged**